Configurations/15-android.conf

```diff
@@ -0,0 +1,207 @@
+#### Android...
+#
+# It takes *one* prior-set environment variable to make it work:
+#
+# ANDROID_NDK=/some/where/android-ndk-<ver>
+#
+# As well as PATH *adjusted* to cover ${CROSS_COMPILE}gcc and company.
+#
+# Note that it's different from original instructions that required to
+# set CROSS_SYSROOT [to $ANDROID_NDK/platforms/android-<api>/arch-<arch>]
+# and CROSS_COMPILE. CROSS_SYSROOT is still recognized [and even required
+# for some legacy targets], but if not set, it's detected and set to the
+# latest Android platform available with appointed NDK automatically. If
```

Comment:
ok for API but how is detected architecture? With variable CROSS_SYSROOT we define both.

Reply:
Originally if you set CROSS_SYSROOT you had to use "matching" target name. For example if you set CROSS_SYSROOT to some/thing/arch-arm, you can't Configure android-x86. Now you'd be able to say Configure android-arm [-D__ANDROID_API__=N] and it will pick proper CROSS_SYSROOT [or die trying].